src/lib/libnoderawfs.js

```diff
@@ -160,7 +166,7 @@ addToLibrary({
     },
     close(stream) {
       VFS.closeStream(stream.fd);
-      if (!stream.stream_ops && --stream.shared.refcnt === 0) {
+      if (!stream.stream_ops && --stream.shared.refcnt <= 0) {
```

Comment:
Isn't it correct to only call the node API when the refcount reaches 0? Further `close()` calls should not call node any more?

Reply:
We have test that closing the same file twice should throw.  Unless we call the underlying API in this case we silently succeed.

Reply:
Oh, I see... yes, this looks right then.